docs/image-cdn.md

```diff
@@ -1,51 +1,68 @@
-Instead of downloading and processing images at build time, it defers processing
-until request time. This can greatly improve build times for sites with remote
-images, such as those that use a CMS. Netlify includes full support for Image
-CDN, on all plans.
-
-When using the image CDN, Gatsby generates URLs of the form
-`/_gatsby/image/...`. On Netlify, these are served by a
-[builder function](https://docs.netlify.com/configure-builds/on-demand-builders/),
-powered by [sharp](https://sharp.pixelplumbing.com/) and Nuxt's
-[ipx image server](https://github.com/unjs/ipx/). It supports all image formats
-supported by Gatsby, including AVIF and WebP.
-
-On first load there will be a one-time delay while the image is resized, but
-subsequent requests will be super-fast as they are served from the edge cache.
+Gatsby Image CDN is a feature available since Gatsby v4.10.0. Instead of
+downloading and processing images at build time, it defers processing until
+request time. This can greatly improve build times for sites with remote images,
+such as those that use a CMS. Netlify includes full support for Image CDN, on
+all plans.
 
 ## Enabling the Image CDN
 
-To enable the Image CDN during the beta period, you should set the environment
-variable `GATSBY_CLOUD_IMAGE_CDN` to `true`.
+To enable the Image CDN, you should set the environment variable
+`NETLIFY_IMAGE_CDN` to `true`. You will also need to declare allowed image URL
+patterns in `netlify.toml`:
 
-Image CDN currently requires the beta version of Gatsby. This can be installed
-using the `next` tag:
-
-```shell
-npm install gatsby@next gatsby-plugin-image@next gatsby-plugin-sharp@next gatsby-transformer-sharp@next
+```toml
+[images]
+remote_images = [
+  'https://example1.com/*',
+  'https://example2.com/*'
+]
 ```
 
-Currently Image CDN supports Contentful and WordPress, and these source plugins
-should also be installed using the `next` tag:
+Exact URL patterns to use will depend on CMS you use and possibly your
+configuration of it.
 
-```shell
-npm install gatsby-source-wordpress@next
-```
+- `gatsby-source-contentful`:
 
-or
+  ```toml
+  [images]
+  remote_images = [
+    # <your-contentful-space-id> will be specified via `spaceId`
+    # gatsby-source-contentful option in gatsby-config
+    "https://images.ctfassets.net/<your-contentful-space-id>/*"
+  ]
+  ```
 
-```shell
-npm install gatsby-source-contentful@next
-```
+- `gatsby-source-drupal`:
+
+  ```toml
+  [images]
+  remote_images = [
+    # <your-drupal-base-url> will be speciafied via `baseUrl`
+    # gatsby-source-drupal plugin option in gatsby-config
+    "<your-drupal-base-url>/*"
+  ]
+  ```
+
+- `gatsby-source-wordpress`:
+
+  ```toml
+  [images]
+  remote_images = [
+    # <your-wordpress-url> will be specified via `url`
+    # gatsby-source-wordpress plugin option in gatsby-config
+    # with "/graphql" endpoint part being omitted
+    "<your-wordpress-url>/wp-content/uploads/*"
```

Comment:
Possibly `<your-wordpress-url>/*` would be good enough here and just easier to setup making it a better choice

Reply:
The wording in the comment is quite confusing

Reply:
/agree

(from https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/getting-started.md)

> In the gatsby-config.js of the starter you just set up, update the plugin options for gatsby-source-wordpress. Change the url option so that it points to your WordPress instance GraphQL url. This should be the full url of your GraphQL endpoint. E.g. https://yoursite.com/graphql

And what I am after is that if URL plugin option is `https://yoursite.com/graphql` then you need to get rid of `/graphql` segment and instead do `/wp-content/uploads/*"` (well maybe just `/*` which I mentioned in initial inline comment) - in any case this needs rewording for sure

Reply:
I think we should recommend just listing the domain/wordpress URL